### PR TITLE
New quick start page

### PR DESCRIPTION
**Pull request by freesig**
This updates the quick start page to be accurate and simpler to use.
I have checked all the command work.
We have moved some of the more dense information into `nix.html` and `vagrant.html`.
It should be easier to follow and get users going quickly.

_Note on not using handlebars_
We couldn't get the repo to clone / build on the on Klaye or my dev machines due to the build files coming from a case sensitive file system.
In the interest of getting this up quickly we opted to just write the `start.html` directly and I have commented out the code which generates it from `start.template.html`. So running the scripts will not affect the `start.html` page and can be used as normal.
I have also changed the link from `/api/VERSION/hdk` to `api/latest/hdk` so it should always be up to date.

### CODE REVIEW

File: nix.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                            <a href="https://proto.holochain.org" title="Holochain Proto Documentation"
+                                target="_blank">Holochain Proto
+                                Documentation</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="col-4">
+                    <ul class="list-unstyled">
+                        <li>
+                            <a href="https://chat.holochain.org/appsup/channels/app-dev" title="Developer Chat"
```
Comment (dhtnetwork) [inline nix.html]: From: <a href="https://chat.holochain.org/appsup/channels/app-dev" title="Developer Chat" Change To: <a href="https://forum.holochain.org/c/technical" title="Developer Forum"
Lets not link to Mattermost anymore.

**Comment by dhtnetwork** [comment]
Tested/confirmed a few times on a Mac (dhtnetwork and Tom)
Tested/no issues on a Linux machine (@mightyalexy200)

**Comment by willemolding** [comment]
Looking great. I love how much more streamlined this. 

Has anyone to got a chance to test on a windows machine yet?

**Comment by dhtnetwork** [comment]
@willemolding Good question. I am testing the steps on the Windows machine.  
FYI @freesig 

- Minor: nix-env  --version   --> message: nix-env (Nix) 2.0   (I guess the Nix version is a bit different on Windows?  I know for sure, on a Mac the nix-version (Nix) 2.2.2
- I was not able to get pass this point. See message attached.  Odd.  

windowsquickstart

windows2

**Comment by freesig** [comment]
@pdaoust yeh check my note on why we had to edit them directly. This PR also disables to the building of start.html so you could continue to build the rest. 
I hope to get a better setup soon with no build artifacts in the repo so this is not an issue anymore.

**Comment by freesig** [comment]
@dhtnetwork I have tried these steps successfully on windows. Everything worked except `hc test`. I'll give it another go tonight though.

**Comment by freesig** [comment]
I wonder if it's worth pushing through this PR so that atleast the mac / linux instructions are accurate. We can always do another one to change the windows instructions. Although I think the steps are correct, it seems like there is something up for holonix windows

**Comment by dhtnetwork** [comment]
@freesig I have scheduled a time for Zippy, you, Klaye, and I to meet to discuss the Quick Start page. Lets address tomorrow.

**Comment by freesig** [comment]
btw @AshantiMutinta looks like you just ran out disk space

**Comment by freesig** [comment]
I've just run this successfully on windows (without git @pdaoust ).
Except for `hc test`. It gives `Error: command npm install --silent was not successful`
Then when run a second time gives:
```
internal/modules/cjs/loader.js:670
    throw err;
    ^

Error: Cannot find module '@holochain/diorama'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/vagrant/my_first_app/test/index.js:4:68)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
Error: command node test/index.js was not successful
```
`hc package` and `hc run -i http` both work though.

Just re the getting stuck, it is ~500mb and if you have a slow connection that might take a while. Also there is over ~1.5 million files to unpack so this might be a challenge for a slower computer especially when running in a VM.
With the space problems I guess this would be not your actual disk space but the VM's allocated space. Does anyone know how to check how much space is allocated to the VM?

**Comment by freesig** [comment]
Ok this is all now up to date with `0.0.29-alpha2`. Happy to merge it now.

**Pull request merged**